namer/namer.cc

```diff
@@ -1186,6 +1186,7 @@ class SymbolDefiner {
 
             // We are on the fast path and there was a previous declaration.
             prior.data(ctx)->resultType = core::Types::todo();
+            prior.data(ctx)->addLoc(ctx, ctx.locAt(field.loc));
```

Comment:
My understanding of this is that it should replace the canonical loc for the field, which should have been in the same file as the field we are finding now?